src/components/Book.js

```diff
@@ -0,0 +1,18 @@
+import propsType from 'prop-types';
+
+const Book = (props) => {
+  const { title, author } = props;
+  return (
+    <div className="p-2">
+      <h2>{title}</h2>
+      <h3>{author}</h3>
+    </div>
```

Comment:
- [ ] @rbhatt1999 great work on your component, however, you forgot to add a delete button, which is part of the requirements
![image](https://user-images.githubusercontent.com/61408860/213389288-ee8ea25c-03e0-48cb-a109-f99d0e27c8eb.png)